releasenotes/notes/defective_qubits-d826ccbd049603b9.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    The `transpile` function supports backends with defective qubits or gates. If a gate/qubit parameter backend
+    property called ``operational`` exists  and it is ``False``, the transpiler will considered the maximun connected
+    subgraph of qubits as a coulpling map and handle the process to avoid using non-operation qubits/gates.
```

Comment:
```suggestion
    Support was added to the :func:`qiskit.compiler.transpile` function for handling backends that have qubits or
    gates marked as defective in their :class:`~qiskit.providers.models.BackendProperties`. If a gate or qubit
    parameter in the :class:`~qiskit.providers.models.BackendProperties` has an ``operational`` field and it
    is set to ``False``, then the :func:`~qiskit.compiler.transpile` function will consider the maximum connected
    subgraph of operational qubits as a coupling map and handle the process to avoid using non-operational
    qubits/gates.
```